ZombieSharp/Statistic.cs

```diff
@@ -0,0 +1,118 @@
+﻿using Dapper;
+using Microsoft.Extensions.Logging;
+using MySql.Data.MySqlClient;
+using Newtonsoft.Json;
+
+namespace ZombieSharp
+{
+    public partial class ZombieSharp
+    {
+        private StatsDatabase statsConfig;
+        MySqlConnection connection = null;
+
+        private void StatsOnLoad()
+        {
+            var statsConfigPath = Path.Combine(ModuleDirectory, $"../../configs/zombiesharp/database.json");
+            statsConfig = JsonConvert.DeserializeObject<StatsDatabase>(File.ReadAllText(statsConfigPath));
+            connection = StatsConnect();
+        }
+
+        private MySqlConnection StatsConnect()
+        {
+            var query = $"Server={statsConfig.hostname}; database={statsConfig.database}; UID={statsConfig.username}; password={statsConfig.password}";
+            var conn = new MySqlConnection(query);
+
+            if(conn == null)
+            {
+                Logger.LogError("[Stats] Connection is null.");
+                return null;
+            }
+
+            conn.Open();
+
+            return conn;
+        }
+
+        private async Task StatsSetData(CCSPlayerController client, int damage = 0, int kill = 0, int infect = 0)
+        {
+            if (client == null)
+            {
+                Logger.LogError("[Stats] client is null.");
+                return;
+            }
+
+            if (connection == null)
+            {
+                return;
+            }
+
+            var clientData = GetClientStatsData(client);
+
+            var query = $"INSERT INTO `zsharp_stats` (player_name, steam_auth, total_dmg, total_kill, total_infect, last_join) " +
```

Comment:
Avoid string interpolations for SQL-queries - it is not safe for SQL-Injections. Redesign it, so it should looks like
```cs
var query = """
    INSERT INTO `zsharp_stats` 
    (player_name, steam_auth, total_dmg, total_kill, total_infect, last_join) 
    VALUES (@PlayerName, @SteamId, @Damage, @Kill, @Infect, @LastJoin)
    """;

var parameters = new 
{
    PlayerName = client.PlayerName,
    SteamId = client.AuthorizedSteamID.SteamId3,
    Damage = damage,
    Kill = kill,
    Infect = infect,
    LastJoin = DateTime.Now // Передаем DateTime объект
};

await dbConnection.ExecuteAsync(query, parameters);
```